nx-X11/programs/Xserver/miext/damage/damage.c

```diff
@@ -1371,12 +1369,14 @@ damageText (DrawablePtr	    pDrawable,
     if (n != 0) {
 	damageDamageChars (pDrawable, pGC->font, x + pDrawable->x, y + pDrawable->y, n,
 			   charinfo, imageblt, pGC->subWindowMode);
+#ifdef NXAGENT_SERVER
 	if (imageblt)
 	    (*pGC->ops->ImageGlyphBlt)(pDrawable, pGC, x, y, n, charinfo,
 				       FONTGLYPHS(pGC->font));
 	else
 	    (*pGC->ops->PolyGlyphBlt)(pDrawable, pGC, x, y, n, charinfo,
 				      FONTGLYPHS(pGC->font));
+#endif
```

Comment:
strictly thinking, this is also GPL-2 license stuff. Even if it is just an ifdef block that gets added.

Reply:
I have taken another look into this and found I have made a mistake. This must be an #ifndef instead of #ifdef. I guess disabling code is ok, license wise. Right?